math/src/field/f62/tests.rs

```diff
@@ -23,7 +23,7 @@ fn add() {
     assert_eq!(BaseElement::from(5u8), BaseElement::from(2u8) + BaseElement::from(3u8));
 
     // test overflow
-    let t = BaseElement::from(BaseElement::MODULUS - 1);
+    let t = BaseElement::new(BaseElement::MODULUS - 1);
```

Comment:
Shouldn't `new()` return a `Result` (and internally run a `try_from()`)? Given that `u64` values over `M` are not valid.

Reply:
One reason for keeping it this way is to provide a fast but "unsafe" method which would allow someone to instantiate a field element when they know for sure that the underlying data is valid. So, given a `u64` we have two options:

- If we don't know whether the underlying data is valid, we should use `try_from()`.
- If we do know that the underlying data is valid, we can use `new()`.

But this behavior should be properly documented - so, I added a note about silent conversion to this method.